vividus-plugin-db/src/main/java/org/vividus/bdd/steps/db/DatabaseSteps.java

```diff
@@ -249,7 +249,8 @@ public void compareData(String sourceSqlQuery, String sourceDbKey, String target
     }
 
     /**
-     * Step waits until data received from SQL request is equal to data from examples table for some duration.
+     * Step waits until data received from SQL request is equal to data from examples table in any order
```

Comment:
Could you please clarify what's unordered? Columns within a row and rows within a result set?

Reply:
and please add System test

Reply:
definition was changed
system test was added